lib/ansible/module_utils/aws/waiters.py

```diff
@@ -151,6 +151,24 @@
                 },
             ]
         },
+        "VpnGatewayDetached": {
+            "delay": 5,
+            "maxAttempts": 40,
+            "operation": "DescribeVpnGateways",
+            "acceptors": [
+                {
+                    "matcher": "path",
+                    "expected": True,
+                    "argument": "VpnGateways[0].State == 'available'",
+                    "state": "success"
+                },
+                {
+                    "matcher": "error",
+                    "expected": False,
```

Comment:
I think you can remove this second dictionary because it won't have any effect. What this would do, if an error happened, is retry if the error code == False (assuming the wait timeout isn't over). But botocore error codes shouldn't be a boolean.